modules/gdscript/tests/scripts/parser/errors/unexpected_token_in_class_body.gd

```diff
@@ -0,0 +1,7 @@
+# Fixes github.com/godotengine/godot/issues/96792
+
+var test = true
```

Comment:
```suggestion
```

Reply:
This line is here intentionally to verify the Parser Error is pointing at the correct line.

Reply:
Ended up reformatting it to achieve the same desired effect

Reply:
Our test system does not allow tracking multiple parser errors. But even without this, we could output line (and column?) for parser errors. This would be better than relying on implicit assumptions. However, this is beyond the scope of this PR, so I suggest not testing it for now.